src/API/Base.php

```diff
@@ -88,13 +90,22 @@ public static function make_request( $endpoint, $method = 'POST', $payload = arr
 		}
 
 		try {
-			$api     = empty( $api ) ? '' : trailingslashit( $api );
+			$api         = empty( $api ) ? '' : trailingslashit( $api );
+			$api_version = 'ads/' === $api ? self::API_ADS_VERSION : self::API_VERSION;
+
 			$request = array(
-				'url'    => self::API_DOMAIN . '/' . $api . 'v' . self::API_VERSION . '/' . $endpoint,
+				'url'    => self::API_DOMAIN . '/' . $api . 'v' . $api_version . '/' . $endpoint,
```

Comment:
Minor: Since we're adjusting this line anyway, this would be more readable with concatenation:

```suggestion
				'url'    => self::API_DOMAIN . "/{$api}v{$api_version}/{$endpoint}",
```